helm/capi-image-builder/values.yaml

```
@@ -34,3 +34,28 @@ ami:
 
 gcp:
   serviceAccountCredentialsJSON: ""
+
+azure:
+  subscriptionId: ""
+  clientId: ""
+  clientSecret: ""
+  tenantId: ""
+  location: westeurope
+  replication_regions: 
```

```suggestion
  replication_regions:
```

Trailing space causing build to fail.